### PR TITLE
An Overview of Jina AI

### LINKED ISSUE #5577: [Machine Learning] An Overview of Jina AI
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

An Overview of Jina AI

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

Jina AI is a cloud-native neural search framework for building state-of-the-art and scalable deep learning search applications in minutes for any kind of unstructured data (videos, images, source code, long text etc.). In this tutorial, I will walk through our readers on how to use this framework to get their ML tasks done. 

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader should be able to understand:
- What is Jina AI?
- Components of Jina AI.
- How to install Jina AI.
- Implement an example to demonstrate how one can use the framework. 

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

Jina AI is a fairly new framework released recently. Many many tutorials are available on it's use. This tutorial will give our readers a high-level overview of the framework and implement an example on how one can use the framework. 

#### References
Please list links to any published content/research that you intend to use to support/guide this article.
- [Jina AI](https://jina.ai/)

### PR DESCRIPTION
**Pull request by wilkistermumbi**
This closes #5577

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by wilkistermumbi** [comment]
This is a duplicate of #5812

**Comment by WanjaMIKE** [comment]
Please fix the plagiarized areas in the attached report.
[New Scan 7_24 AM Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/8107194/New.Scan.7_24.AM.Copyleaks.Report.pdf)

File: content/articles/an-overview-of-jina-ai/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/an-overview-of-jina-ai/hero.png 
+    alt: An Overview of Jina AI Example Image
+---
+Searching for information on the web is not new. We use search every day on Twitter, LinkedIn, Stack Overflow, and Amazon. 
+<!--more-->
+The most common way we've all been searching for a long time now is with the Google Search platform. But, that's now what we will be discussing today, we will be discussing neural search and how it is different from the standard search that we are used to. Besides, the tutorial will walk you through how to use the Jina AI framework to get your ML tasks done.
```
Comment (Bayler) [inline content/articles/an-overview-of-jina-ai/index.md:19]:
```suggestion
The most common way we've all been searching for a long time now is with the Google Search platform. But, that's not what we will be discussing today. We will be discussing neural search and how it is different from the standard search that we are accustomed to. Besides, the tutorial will walk you through how to use the Jina AI framework to get your ML tasks done.
```

File: content/articles/an-overview-of-jina-ai/index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+- [Fundamental concepts](#fundamental-concepts)
+- [Implementing an example to demonstrate how one can use the framework](#implementing-an-example-to-demonstrate-how-one-can-use-the-framework)
+- [Wrapping-up](#wrapping-up)
+- [Further reading](#further-reading)
+
+### What is neural search 
+Before neural search, developers had to write every single instruction to help an application retrieve information. This process was time-consuming and would give developers headaches as they tried to develop these applications. But this isn't the case with neural search. With the advent of neural networks, the way developers write rules changed. One could easily train a neural network to perform a task and the network gets better with the more data it sees. This is the same case with neural search. In simple words, it is bringing neural networks to search.
+
+Pre-trained neural networks are deployed to retrieve information. These networks are trained to retrieve information and get better at information retrieval when fed with a lot of data. Jina AI is one type of neural search framework that uses deep neural networks to perform the search.
```
Comment (Bayler) [inline content/articles/an-overview-of-jina-ai/index.md:38]:
```suggestion
Before neural search, developers had to write every instruction to help an application retrieve information. This process is time-consuming and would give developers headaches as they try to develop these applications. But this isn't the case with neural search. With the advent of neural networks, the way developers write rules changed. One could train a neural network to perform a task, and the network gets better with the more data it sees. It is the same case with neural search. In simple words, it is bringing neural networks to search.

Pre-trained neural networks are deployed to retrieve information. These networks are trained to retrieve information and get better at information retrieval when fed with a lot of data. Jina AI is a neural search framework that uses deep neural networks to perform the search.
```

File: content/articles/an-overview-of-jina-ai/index.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+### What is Jina AI
+Jina AI is an open-source, cloud-native neural search framework. It is used for building state-of-the-art and scalable deep learning search applications for any kind of modality. For example, videos, images, source code, long text, etc. The framework allows you to import a "lightweight" version of the Google search engine into your project.     
+
+It was first introduced in May 2020 by [Dr. Han Xiao](https://hanxiao.io/about/). He is also the creator of well-known open-source projects such as [bert-as-a-service](https://github.com/hanxiao/bert-as-service) and the popular [fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) dataset. Currently, the framework is maintained by [Jina AI](https://jina.ai/), an open-source tech startup based in Berlin, Germany.
```
Comment (Bayler) [inline content/articles/an-overview-of-jina-ai/index.md:43]:
```suggestion
It was first introduced in May 2020 by [Dr. Han Xiao](https://hanxiao.io/about/). He is also the creator of well-known open-source projects such as [bert-as-a-service](https://github.com/hanxiao/bert-as-service) and the popular [fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) dataset. Currently, [Jina AI](https://jina.ai/), an open-source tech startup based in Berlin, Germany, maintains the framework.
```

File: content/articles/an-overview-of-jina-ai/index.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+### How to install Jina AI
+The Jina AI framework is easy to set up using a quick `pip` install as shown below:
```
Comment (Bayler) [inline content/articles/an-overview-of-jina-ai/index.md:46]:
```suggestion
The Jina AI framework is easy to set up using a quick `pip` install, as shown below:
```

File: content/articles/an-overview-of-jina-ai/index.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+#### Executor
+It processes the data. In this case, our data comes from the `Document`.
+
+#### Flow
+The Flow streamlines and distributes the `Executors`.
+
+1. It consists of pods. They are the "brains" of Jina. These pods help us achieve specific tasks such as segmenting, encoding, and ranking.
+2. Context manager.
+3. Abstraction of high-level tasks i.e., index or query.
```
Comment (Bayler) [inline content/articles/an-overview-of-jina-ai/index.md:70]:
```suggestion
1. It consists of pods. They are the "brains" of Jina. These pods help us achieve specific tasks, such as segmenting, encoding, and ranking.
2. Context manager.
3. Abstraction of high-level tasks, i.e., index or query.
```

**Comment by Bayler** [comment]
@wilkistermumbi Some minor edits before we can move for final review.

**Comment by wilkistermumbi** [comment]
@Bayler Done.

**Comment by Kayere** [comment]
Great article @wilkistermumbi.
Thank you for the review @Bayler.

**Pull request merged**